operators/advanced_network/advanced_network/managers/gpunetio/adv_network_doca_mgr.cpp

```diff
@@ -1209,11 +1209,9 @@ int DocaMgr::rx_core(void* arg) {
   }
   pthread_setname_np(self, "RX_WORKER");
 
-// WAR for Holoscan management of threads.
-// Be sure application thread finished before launching other CUDA tasks
-#if RX_PERSISTENT_ENABLED == 1
-  sleep(5);
-#endif
+  // WAR for Holoscan management of threads.
+  // Be sure application thread finished before launching other CUDA tasks
+  sleep(2);
```

Comment:
Should eventually get rid of this, but it's outside the scope

Reply:
100, will create jira ticket